plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -393,6 +433,33 @@ protected List<CastTestCase> supportedCastTypePushdown()
                 new CastTestCase("c_numeric_30_2", "integer", "c_integer"),
                 new CastTestCase("c_decimal_negative", "integer", "c_integer"),
 
+                new CastTestCase("c_char_50", "char(10)", "c_char_10"),
+                new CastTestCase("c_bpchar", "char(10)", "c_char_10"),
+
+                new CastTestCase("c_char", "char(1)", "c_nchar"),
+                new CastTestCase("c_char_10", "char(50)", "c_char_50"),
+                new CastTestCase("c_char_10", "char(256)", "c_bpchar"),
+                new CastTestCase("c_char", "char(4096)", "c_char_4096"),
+
+                new CastTestCase("c_varchar_50", "varchar(10)", "c_varchar_10"),
+                new CastTestCase("c_nvarchar_50", "varchar(10)", "c_nvarchar_10"),
+                new CastTestCase("c_text", "varchar(10)", "c_varchar_10"),
+
+                new CastTestCase("c_nvarchar_10", "varchar(50)", "c_nvarchar_50"),
+                new CastTestCase("c_varchar_10", "varchar(50)", "c_text"),
+                new CastTestCase("c_varchar_10", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_varchar_10", "varchar(65535)", "c_varchar_50"),
+                new CastTestCase("c_varchar_10", "varchar(256)", "c_nvarchar"),
+                new CastTestCase("c_varchar_10", "varchar(10)", "c_nvarchar_10"),
+                new CastTestCase("c_varchar_10", "varchar(50)", "c_nvarchar_50"),
+                new CastTestCase("c_varchar_10", "varchar(65535)", "c_nvarchar_65535"),
+                new CastTestCase("c_varchar_10", "varchar(256)", "c_text"),
+                new CastTestCase("c_varchar_10", "varchar", "c_text"),
+
+                new CastTestCase("c_varchar_unicode", "varchar", "c_varchar_50"),
```

Comment:
Can we add some more test for casting to `varchar` like from `c_varchar_10`, `c_nvarchar_unicode`, `c_text`, `c_nvarchar_unicode`?